forms/models/elements/subform.js

```diff
@@ -195,9 +195,6 @@ define(function (require) {
                 return memo;
               }
 
-              // let the element know who its parent is
-              element.parentElement = self;
```

Comment:
arg, there should be a modification on line 203 to set this on the actual element.